tests/modules/nf-core/samtools/cramsize/test.yml

```diff
@@ -0,0 +1,13 @@
+## TODO nf-core: Please run the following command to build this file:
+#                nf-core modules create-test-yml /cramsize
```

Comment:
Can you generate the test yaml file?

Reply:
Thanks for all your advice @nvnieuwk ! I have just added a stub section and a test.yml file. I will be very grateful if you can review it, please.

Thanks a lot in advance!